src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/index.html.twig

```diff
@@ -50,10 +50,40 @@
       </div>
     {% endblock %}
   {{ form_end(generalForm) }}
-{% endblock %}
 
-{% block javascripts %}
-  {{ parent() }}
+  {{ form_start(passwordPolicyForm, {attr : {class: 'form'}, action: path('admin_security_password_policy_save') }) }}
+    {% block administration_form_password_policy %}
+      <div class="card" id="configuration_fieldset_password_policy">
+        <h3 class="card-header">
+          <i class="material-icons">settings</i> {{ 'Password policy'|trans({}, 'Admin.Global') }}
+        </h3>
+        <div class="card-body">
+          <div class="form-group row">
+            {{ form_label(passwordPolicyForm.minimum_score) }}
+            <div class="col-sm">
+              {{ form_widget(passwordPolicyForm.minimum_score) }}
+              <small class="form-text">
+                {{ 'Scores are integers from 0 to 4.'|trans({}, 'Admin.Advparameters.Feature') }}
+                <ol id="help-password-score">
+                  <li>{{ '0 means the password is extremely guessable (within 10^3 guesses), dictionary words like "password" or "mother" score a 0.'|trans({}, 'Admin.Advparameters.Feature') }}</li>
+                  <li>{{ '1 is still very guessable (guesses &lt; 10^6), an extra character on a dictionary word can score a 1'|trans({}, 'Admin.Advparameters.Feature') }}</li>
+                  <li>{{ '2 is somewhat guessable (guesses &lt; 10^8), provides some protection from unthrottled online attacks'|trans({}, 'Admin.Advparameters.Feature') }}</li>
```

Comment:
```suggestion
                  <li>{{ '2 is pretty easy to guess (guesses lesser than 10^8). It provides some protection from unthrottled online attacks.'|trans({}, 'Admin.Advparameters.Feature') }}</li>
```

Reply:
And just to be sure: Are we talking about seconds? Like it takes 10^8 seconds to guess? I'm not sure to understand what is the parenthesis. 🤔